shared/chat/common/selectable-big-team-channel.js

```diff
@@ -69,6 +69,8 @@ class FilterBigTeamChannel extends PureComponent<Props, State> {
   }
 }
 
+const smallRowHeight = isMobile ? 64 : 56
```

Comment:
we cannot disconnect this code from RowSizes or the virtualized layout of the inbox will get broken. its one shared value